code/datums/dna/dna.dm

```diff
@@ -490,19 +490,20 @@ GLOBAL_LIST_INIT(total_uf_len_by_block, populate_total_uf_len_by_block())
 	if(!has_dna())
 		return
 
-/mob/living/carbon/human/updateappearance(icon_update = TRUE, mutcolor_update = FALSE, mutations_overlay_update = FALSE)
+/mob/living/carbon/human/updateappearance(icon_update = TRUE, mutcolor_update = FALSE, mutations_overlay_update = FALSE, datum/dna/passed_dna)
```

Comment:
Wait why is this argument not in the base definition `mob/living/carbon/proc/updateappearance` anymore?